src/xcode/ENA/ENA/Source/Models/ClientMetaData/ClientMetadata.swift

```diff
@@ -0,0 +1,56 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+struct ClientMetadata: Codable {
+	var cwaVersion: Version?
+	var iosVersion: Version
+	var eTag: String?
```

Comment:
ETag for…? This should be commented for which request/response we store this.

Reply:
the eTag for the last fetched appConfiguration, I will add a comment